src/components/App.tsx

```diff
@@ -17,14 +18,48 @@ function App() {
   // State for the line height on the left
   const [lineHeightLeft, setLineHeightLeft] = useState<number>(1.5);
 
+  // Opentype feature option names from the gsub table of the font file on the left
+  const [fontFeatureOptionsLeft, setFontFeatureOptionsLeft] = useState<unknown[]>([]);
+
+  // Opentype feature option names from the gsub table of the font file on the right
+  const [fontFeatureOptionsRight, setFontFeatureOptionsRight] = useState<unknown[]>([]);
+
   // Handler for when a font is selected on the left side
   const handleFontSelectedLeft = (selectedFont: File | null) => {
     setSelectedFontLeft(selectedFont);
+    if (selectedFont != null) {
```

Comment:
A bulk of this logic could be extracted away into another method as it essentially is the same for left and right but for now, it looks pretty good!

Reply:
I agree, if this is pulled through then a refactor would be needed.